tqec/main.py

```diff
@@ -23,7 +24,21 @@ def root():
     # App Engine itself will serve those files as configured in app.yaml.
     app.run(host="127.0.0.1", port=8080, debug=True)
 
-@app.route("/stim", methods=['GET'])
+@app.route("/stim", methods=['GET', 'POST'])
 @cross_origin(supports_credentials=True)
-def pdf():
-    return send_file("teleportation.stim")
\ No newline at end of file
+def jsonToStim():
+    _json = request.get_json()
+    if _json is None:
+        return "No JSON provided", 400
+    # Construct the plaquettes from the given file
+    plaquettes = []
+    for plaquette in _json["plaquettes"]:
+        # Construct the qubits
+        qubits = [PlaquetteQubit(Position(qubit["x"], qubit["y"])) for qubit in plaquette["qubits"]]
```

Comment:
+1 to that. Have you considered a library like Pandas?

Reply:
Not really, what does pandas have to do with that?

Reply:
I was suggesting using a (de-)serialization library for that, but I'll leave it up to you.

Reply:
I might try to use [`pydantic`](https://docs.pydantic.dev/latest/), because it seems to be the right tool and I want to learn about it, but a simple use of the standard `json` package and a few (~10) methods should be enough and will likely be:
1. easier and quicker to write,
2. easier to understand for newcomers, 
3. enough for our current purpose.

I am open to discussion on that part though, I never really made a JSON export/import interface to be used by external languages / other developers than myself, so I am a beginner on that subject.